test/e2e/storage/csi_objects.go

```diff
@@ -366,10 +367,19 @@ func csiHostPathPod(
 		return nil
 	}
 
-	ret, err := podClient.Create(pod)
-	if err != nil {
-		framework.ExpectNoError(err, "Failed to create %q pod: %v", pod.GetName(), err)
-	}
+	// Creating the pod can fail initially while the service
+	// account's secret isn't provisioned yet ('No API token found
+	// for service account "csi-service-account", retry after the
+	// token is automatically created and added to the service
+	// account', see https://github.com/kubernetes/kubernetes/issues/68776).
+	// We could use a DaemonSet, but then the name of the csi-pod changes
+	// during each test run. It's simpler to just try for a while here.
+	var ret *v1.Pod
+	Eventually(func() error {
```

Comment:
May be worth adding a CreatePodWithRetry utility function?

Reply:
Yes, that may be useful. I'll do that.

Reply:
Done. I ended up adding CreateEventually to pod.go's PodClient, which looked like the right place because it already had a Create (without retry).